data/models/users.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../dbConfig");
+
+module.exports = {
+  getUsers,
+  login,
+  register
+};
+
+function getUsers() {
+  return db("users");
+}
+
+function login(name) {
+  return db("users").where(name);
+}
```

Comment:
I presume the `name` is an object. It's not intuitive.

Reply:
ammended